ui/dashboards/src/components/Dashboard/Dashboard.stories.tsx

```diff
@@ -213,3 +218,241 @@ export const CustomEmptyState: Story = {
     },
   },
 };
+
```

Comment:
This data matches what we use in http://localhost:3000/projects/testing/dashboards/TimeSeriesChartPanel?start=6h for e2e tests as an easy to set up starter case for comparison.